rts/motoko-rts/src/types.rs

```diff
@@ -921,6 +921,10 @@ pub(crate) unsafe fn block_size(address: usize) -> Words<u32> {
 
         TAG_BIGINT => {
             let bigint = address as *mut BigInt;
+            if (*bigint).mp_int.used != (*bigint).mp_int.alloc {
+                println!(100, "used:{}, alloc: {}", (*bigint).mp_int.used, (*bigint).mp_int.alloc);
+            }
+            (*bigint).mp_int.alloc = (*bigint).mp_int.used;
```

Comment:
stats are in https://github.com/dfinity/motoko/pull/4102/files/e15afc367f58f61db4756f6f41e513890d7c8163#r1253257503 and also past commits...